docker/elipselink/README.md

```diff
@@ -0,0 +1,51 @@
+# Polaris with EclipseLink, Postgres and Spark SQL
+
+1. If such an image is not already present, build the Polaris image with support for EclipseLink and
+   the Postgres JDBC driver:
+
+    ```shell
+    ./gradlew clean :polaris-quarkus-server:assemble :polaris-quarkus-admin:assemble \
+       -PeclipseLinkDeps=org.postgresql:postgresql:42.7.4 \
+       -Dquarkus.container-image.build=true
```

Comment:
nit: would it make sense to override the image name/tag to indicate it's with PostgreSQL?